website/components/mdx/_configure.mdx

```diff
@@ -2,7 +2,7 @@ import { Callout } from 'nextra-theme-docs'
 
 ## Configure Destination Plugin
 
-Destination plugins define where you will be syncing your data to (See: [available destinations](/docs/plugins/destinations)).
+Destination plugins define where you will be syncing your data to (See: [available destinations](/docs/plugins/destinations/overview)).
```

Comment:
This is technically not needed as we have a redirect, but saves users the extra hop